From 0546dae565b02472fe29c67a1039db9eaf4dc2c7 Mon Sep 17 00:00:00 2001
From: Ryan Lerch <ryanlerch@users.noreply.github.com>
Date: Fri, 1 Aug 2014 10:48:45 -0400
Subject: [PATCH] Update org.baedert.corebird.appdata.xml.in

Updated the licence tag to metadata licence (appdata xml spec change), and added a summary
---
 data/org.baedert.corebird.appdata.xml.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/data/org.baedert.corebird.appdata.xml.in b/data/org.baedert.corebird.appdata.xml.in
index 4fccba7..fdfc84a 100644
--- a/data/corebird.appdata.xml.in
+++ b/data/corebird.appdata.xml.in
@@ -2,8 +2,9 @@
 <!-- Copyright 2013 Ryan Lerch <ryanlerch@fedoraproject.org> -->
 <application>
  <id type="desktop">org.baedert.corebird.desktop</id>
- <licence>CC0</licence>
- <description>
+ <summary>twitter client</summary>
+	<metadata_license>CC0-1.0</metadata_license> 
+	<description>
   <_p>
    Corebird is a native GTK+ twitter client that provides vital features such as Direct Messages (DMs), tweet notifications, conversation views.
   </_p>
-- 
2.0.3

